assets/src/components/map.tsx

```diff
@@ -226,7 +228,26 @@ const LeafletShape = ({ shape }: { shape: Shape }) => {
           center={[stop.lat, stop.lon]}
           radius={3}
         >
-          <Popup className="m-vehicle-map__stop-tooltip">{stop.name}</Popup>
+          <Popup className="m-vehicle-map__stop-tooltip">
+            {stop.name}
+            {inTestGroup(MAP_BETA_GROUP_NAME) && (
+              <>
+                <br></br>
+                <a
```

Comment:
Suggestion: now that https://github.com/mbta/skate/pull/1809 is merged, this could use the [`StreetViewButton`](https://github.com/mbta/skate/blob/master/assets/src/components/streetViewButton.tsx) for consistency with the vehicle properties card!